metadata.yaml

```diff
@@ -0,0 +1,26 @@
+---
+project:
+  name: serverless-operator
+  version: 1.10.0
+
+olm:
+  replaces: 1.9.0
+  skipRange: '>=1.9.0 <1.10.0'
+  channels:
+    default: '4.3'
+    list:
+      - '4.5'
+
+requirements:
+  kube:
+    minVersion: 1.15.0
+  golang: 1.14
+
+components:
+  serving: 0.16.0
+  eventing: 0.16.0
+  cli: 0.16.1
+  kourier: 1.1.0
```

Comment:
This needs to be `maistra` or something like that. `kourier` needs to be 0.16.0 in this case.

Reply:
This is used in https://github.com/cardil/serverless-operator/blob/5caa536/hack/generate/csv.sh#L24 

I'm ok to changing that key to `maistra`, but I don't understand the second part of your comment: 

> kourier needs to be 0.16.0 in this case.

Reply:
Kourier is a separate component that is not necessarily at serving's version.

Reply:
Are you talking about this repo: https://github.com/knative-sandbox/net-kourier? Serverless Operator doesn't refer to it, isn't? Changing this to maistra will remove the confusion.

Reply:
Yes we do reference it, via https://github.com/openshift-knative/net-kourier/

Reply:
Done.